feat: Rebrand cleanup

ariansobczak-rst opened the pull request:
Cleaned up rebranding from SoftwareBrothers

- updated logo 
- changed logo and link in **Need more advanced solutions** section
- replaced SoftwareBrothers "heart" component with MadeWithLove

![image](https://user-images.githubusercontent.com/85543813/157879456-4c5530b1-2e43-4bf3-8e25-e40e520a2def.png)

dziraf reviewed (approved):
Technically, we don't need to support `softwareBrothers` option anymore if we release this as a breaking change in v6

ariansobczak-rst commented:
> Technically, we don't need to support `softwareBrothers` option anymore if we release this as a breaking change in v6

Removed all `softwareBrothers` related things. Introduced Migration guide.
